### PR TITLE
Include additional UI variables in docs

### PR DESCRIPTION
**Pull request by webb-ben**
Includes additional environment variables to the docs as implemented in https://github.com/wmo-im/wis2box-ui/pull/95

Addresses:
https://github.com/wmo-im/wis2box-ui/issues/93
https://github.com/wmo-im/wis2box-ui/issues/98

### CODE REVIEW

**Comment by webb-ben** [comment]
Ready for review after: https://github.com/wmo-im/wis2box-ui/pull/103

**Comment by tomkralidis** [comment]
FYI https://github.com/wmo-im/wis2box-ui/pull/103 is now merged.  Let me know when ready for review/out of draft.

File: wis2box-create-config.py [hunk 1]
```diff
@@ -284,6 +284,39 @@ def get_wis2box_url() -> str:
...
+    Prompt the user to enter a custom UI logo path or URL.
+
+    :returns: string of URL logo
+    """
+    while True:
+        logo = input("Enter the URL for the custom UI logo (leave blank to skip): ").strip() # noqa
+        msg = f"Confirm custom UI logo: '{logo or 'Default'}'? (y/n): "
+        confirm = input(msg).strip().lower()
+        if confirm == 'y':
+            return logo if logo else None
```
Comment (tomkralidis) [inline wis2box-create-config.py:298]:
```python
return logo or None
```

File: wis2box-create-config.py [hunk 2]
```diff
@@ -284,6 +284,39 @@ def get_wis2box_url() -> str:
...
+
+
+def get_default_ui_language():
```
Comment (tomkralidis) [inline wis2box-create-config.py:301]:
```python
def get_default_ui_language() -> Union[str, None]:
```

Note that you will need to update `from typing import Tuple` to `from typing import Tuple, Union` on line 29.

File: docs/source/reference/configuration.rst [hunk 3]
```diff
@@ -169,14 +169,17 @@ Restart the mosquitto container for the changes to take effect with the command:
 Web application
 ^^^^^^^^^^^^^^^
 
-Web application configuration provides the ability to customize web components.
+Web application configuration provides the ability to customize web components. All of these are optional.
```
Comment (tomkralidis) [inline docs/source/reference/configuration.rst:172]: Web application configuration provides the ability to customize web components. All of the below directives are optional.

**Review by tomkralidis — CHANGES_REQUESTED** [review]

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**